sites/docs/src/content/docs/contributing/project_proposals.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Project proposals / RFCs"
+subtitle: Process for proposing major new nf-core projects
+---
+
+# Summary
+
+1. Got a big idea? Post it on [`#rfc-suggestions`](https://nfcore.slack.com/archives/C08TXM0GGMT) for quick feedback.
+2. Create an issue on nf-core/proposals using the RFC template, explaining **_what_** you want to do.
+3. Once approved by the core team, open a PR with a full RFC text as markdown, extending the issue with details on **_how_** it should be implemented.
+4. Start development and update the RFC PR as needed as you go along.
+5. Once the new feature ships, the RFC PR is merged and the markdown used for future reference.
+
+# Preamble
+
+The nf-core ‘Request for Comment’ (RFC) process is designed to give the community a voice and visibility to large projects that affect the entire community.
+
+## Glossary
+
+<dl>
+  <dt>RFC (Request for Comment):</dt>
+  <dd>A formal proposal submitted for discussion, typically involving significant changes or new features. An RFC outlines the motivation, requirements, and steps necessary for implementation, and invites feedback and collaboration from the community before a final decision is made.</dd>
+
+  <dt>Proposal champion:</dt>
+  <dd>An individual who takes ownership of advancing a proposal through the RFC process. This role is self-nominated and open to anyone, including both project maintainers and other community members. The champion may be the original author of the proposal or someone who joins later. Responsibilities include drafting the detailed RFC, managing and integrating community feedback, and helping to guide the implementation of the proposal.</dd>
+</dl>
+
+:::note
+**You are not alone as a champion!** If this is your first time writing an RFC or design document, our maintainer team will be available to work with you and guide you through this process.
```

Comment:
```suggestion
**You are not alone as a champion!** If this is your first time writing an RFC or design document, our maintainers team will guide you through this process.
```

I propose to reduce the scope and add this new responsibility in the `#responsibilities` of `maintainers` in the `governance` page.

Reply:
Sounds good to me